spec/models/navigable_name_updater_spec.rb

```diff
@@ -14,8 +13,13 @@
       expect { navigable_name_updater.update }.to change { ImageSearchLabel.where(name: 'Assit').count }.from(0).to(1)
     end
 
-    it 'updates all video search labels except for English/Spanish sites' do
-      expect { navigable_name_updater.update }.to change { RssFeed.where(name: 'Test Entry').count }.from(0).to(1)
+    context 'when a video feed label is outdated' do
+      before { video_feed.update!(name: 'outdated') }
+
+      it 'updates all video search labels except for English/Spanish sites' do
+        expect { navigable_name_updater.update }.
+          to change { video_feed.reload.name }.from('outdated').to('Videos')
+      end
```

Comment:
I've updated this spec minimally to make it pass. I don't love the update, but it was a weird spec to begin with. The code itself is walking dead, so I opted not to burn any more time on refactoring the spec.